src/main/java/frc/robot/Robot.java

```diff
@@ -92,25 +87,24 @@ public void autonomousInit() {
 
     /** This function is called periodically during autonomous. */
     @Override
-    public void autonomousPeriodic() {
-        switch (m_autoSelected) {
-            case kCustomAuto:
-                // Put custom auto code here
-                break;
-            case kDefaultAuto:
-            default:
-                // Put default auto code here
-                break;
-        }
-    }
+    public void autonomousPeriodic() {}
 
     /** This function is called once when teleop is enabled. */
     @Override
     public void teleopInit() {}
 
     /** This function is called periodically during operator control. */
     @Override
-    public void teleopPeriodic() {}
+    public void teleopPeriodic() {
+        double leftY = driverController.getLeftY();
+        double rightY = driverController.getRightY();
+
+        driveLeftParent.set(leftY);
+        driveRightParent.set(rightY);
+
+        driveLeftChild.set(leftY);
+        driveRightChild.set(rightY);
```

Comment:
This will be changed to a leader-follower system later. (We did not teach the new people about it yet.)